frontend/components/ui/StreamingJsonlRenderer.tsx

```
@@ -267,11 +290,29 @@ export function StreamingJsonlRenderer({
+            <div className="leading-6 my-2 select-text text-foreground">
+              <div className="flex items-start justify-between gap-3">
+                <div className="flex-1">
+                  {lead && (
+                    <span className="font-semibold text-foreground">
+                      {lead}:{" "}
+                    </span>
+                  )}
+                  <span>{content}</span>
+                </div>
+                {hasReferences && (
+                  <EnhancedReferenceIndicator
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)